app/src/main/java/edu/wisc/ece/pinpoint/pages/pins/PinViewFragment.java

```diff
@@ -47,6 +59,16 @@ public class PinViewFragment extends Fragment {
     private ConstraintLayout metadataBar;
     private String pid;
     private String authorUID;
+    private RecyclerView commentRecyclerView;
+    private ImageView addCommentButton;
+    private TextInputLayout addCommentInputLayout;
+    private TextInputEditText addCommentEditText;
+    private ImageButton backButton;
+    private ImageView sendCommentButton;
+    private NestedScrollView scrollView;
+
+    private final int COMMENT_FOCUS_SCROLL = 1000;
+    private ArrayList<Comment> comments;
```

Comment:
Can we make this a `List<Comment>` type so it matches the signature of the `firebase.fetchComments` method?